### ISSUE TITLE
[DLNA] Many log messages from Media Renderers trying to see old events

### ISSUE THREAD

**Issue by mooninite** [jellyfin/jellyfin#3104]
**Describe the bug**
I have a Yamaha receiver that acts as a DLNA Media Render device and it is causing a lot of log output after I restart Jellyfin. The cause appears to be the receiver is looking for old events that no longer exist after restarting Jellyfin. Can we either stop logging these errors or is there a response that could be sent to the receiver so it stops looking?

**System (please complete the following information):**
 - OS: Fedora 32
 - Virtualization: None
 - Clients: Browser, DLNA
 - Browser: Firefox 76
 - Jellyfin Version: 10.5.5
 - Playback: DLNA
 - Installed Plugins: Playback Reporting
 - Reverse Proxy: apache
 - Base URL: none
 - Networking: Host
 - Storage: local

**To Reproduce**

1. Start Jellyfin
2. Plug in receiver into power outlet
3. No log output
4. Restart Jellyfin
5. Logs start showing DLNA errors.

A "fix" is to unplug the receiver and plug it back in as the receiver is always "on" even when it is "off" as it is connected to Wi-fi listening for my controller events to remotely power it on/off. After that the log messages stop until I restart Jellyfin.

**Expected behavior**

No log output.

**Logs**

(repeated every 3 seconds...)
```
[15:30:19] [ERR] [61] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request. URL: http://10.0.x.x:8096/dlna/<UUID redacted>/contentdirectory/events
 System.NullReferenceException: Object reference not set to an instance of an object.
    at Emby.Dlna.Eventing.EventManager.RenewEventSubscription(String subscriptionId, String notificationType, String requestedTimeoutString, String callbackUrl)
    at Emby.Dlna.Api.DlnaServerService.ProcessEventRequest(IEventManager eventManager)
    at Emby.Server.Implementations.Services.ServiceExecGeneral.Execute(Type serviceType, IRequest request, Object instance, Object requestDto, String requestName)
    at Emby.Server.Implementations.Services.ServiceController.Execute(HttpListenerHost httpHost, Object requestDto, IRequest req)
    at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken)
    at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```

**Screenshots**

**Additional context**

**Comment by PrplHaz4** [comment]
at the very least this should be handled by not throwing an exception i think

**Comment by BaronGreenback** [comment]
Null pointer operation. Fixed in #3337.
